pkgs/os-specific/linux/kernel/common-config.nix

```diff
@@ -394,6 +394,10 @@ let
       EXT4_FS_SECURITY  = yes;
       EXT4_ENCRYPTION   = option yes;
 
+      NTFS_FS = no;
```

Comment:
```suggestion
      NTFS_FS = no;
      NTFS3_FS = module;
```

Reply:
It's already implied for `NTFS3_FS` to be a module (same for every other filesystem there).